diff --git a/docroot/.htaccess b/docroot/.htaccess
index 9a73a3d3a..d7006c7b7 100644
--- a/docroot/.htaccess
+++ b/docroot/.htaccess
@@ -187,3 +187,16 @@ AddEncoding gzip svgz
   # Disable Proxy header, since it's an attack vector.
   RequestHeader unset Proxy
 </IfModule>
+
+# This bit of text has been added using a patch. Which is, for now,
+# the best way of ensuring changes to your .htaccess file do not get
+# overridden by core's new scaffolding plugin.
+# See etc/scaffold-modifications/htaccess.patch for the source of this text.
+# To update the patch file, place the newest "pure", untouched .htaccess file
+# in your docroot, stage the file, apply your changes, do NOT stage the file
+# again and then run the following command from above docroot:
+# "git diff docroot/.htaccess > etc/scaffold-modifications/htaccess.patch"
+# Afterwards, run composer update again to apply your changes properly.
+#
+# @see https://www.drupal.org/project/drupal/issues/3092563
+# @see https://www.drupal.org/project/drupal/issues/3095214
